cpp/velox/operators/plannodes/RowVectorStream.h

```diff
@@ -59,7 +59,7 @@ class ValueStreamNode : public facebook::velox::core::PlanNode {
     return outputType_;
   }
 
-  const std::vector<facebook::velox::core::PlanNodePtr>& sources() const {
+  const std::vector<facebook::velox::core::PlanNodePtr>& sources() const override {
```

Comment:
nit: curious why this format could pass format check?

Reply:
not sure, I just apply `clang-format -i`